issues.css

```diff
@@ -0,0 +1,46 @@
+body {
+    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
Can this file be placed in a static folder?
[nitpick] This repository has to do better job at code organization. It needs cleaning and reorganizing in folder structure.

Reply:
The generated html cannot reference the css file within the static folder. We can restructure it like this
```bash
project-root
  - html
    - css
    - template
    - generated
```

Reply:
Sounds like a good idea.